### DISCUSSION TITLE
MacOS support

### DISCUSSION THREAD

**Discussion by jmagdowski** [the-marenga/sf-scrapbook-helper#37] [Q&A] [answered]
Hey! Has development of the macOS version been abandoned? :(

**Comment by the-marenga** [comment, marked as answer]
No, don't worry, I still mainly develop this on mac myself. The auto-build system I use to provide the binaries on the release page just doesn't work with the mac version anymore, which I found out only after the release. I am going to upload a mac version manually from now on, but I haven`t had the time yet, as after this release, I was fixing the build system for windows & linux, which is the main userbase and just got back to my machine a few minutes ago. You can expect a binary mac download in the next few minutes
